test/utils.ts

```diff
@@ -1,4 +1,4 @@
-import diff from 'jest-diff';
+import * as diff from 'jest-diff';
```

Comment:
What is the purpose of this change? Also, I'm getting this error when trying to run the tes, but If I revert this change, It works. Could you help out with this?

This is a screenshot of the error
<img width="1425" alt="Screenshot 2019-07-12 at 12 00 35 PM" src="https://user-images.githubusercontent.com/3120013/61123757-132c4580-a49d-11e9-9039-15043df9c6eb.png">

Reply:
![image](https://user-images.githubusercontent.com/5889406/61131129-f3237300-a4ea-11e9-9326-676b1ccaa548.png)
Because this,I will revert it. 
But i can run with `import * as diff from 'jest-diff';`